app/components/Article/Contents.tsx

```diff
@@ -123,9 +123,9 @@ const insertGlossary = (pageid: string, glossary: Glossary) => {
       addPopup(
         e as HTMLSpanElement,
         `glossary-${entry.term}`,
-        `<div class="glossary-popup flex-container black col-6 small">
+        `<div class="glossary-popup flex-container black small">
               <div class="contents col-8">
-                   <h3>${entry.term}</h3>
+                   <div class="small-bold">${entry.term}</div>
```

Comment:
@melissasamworth you had `p` on the bug report, but that has auto padding set from the underlying article, so I left divs

Reply:
That's fine. In the future perhaps we can create a `padding: 0` class to add or something